chain/client/src/client.rs

```diff
@@ -186,8 +179,12 @@ pub struct Client {
     /// When the "sync block" was requested.
     /// The "sync block" is the last block of the previous epoch, i.e. `prev_hash` of the `sync_hash` block.
     pub last_time_sync_block_requested: Option<DateTime<Utc>>,
-
+    /// Helper module for stateless validation functionality like chunk witness production, validation
+    /// chunk endorsements tracking etc.
     pub chunk_validator: ChunkValidator,
+    /// Tracks current chunks that are ready to be included in block
+    /// Manages filtering out chunks that maybe from banned chunk_producers or similar reasons
```

Comment:
```suggestion
    /// Also tracks banned chunk producers and filters out chunks produced by them
```
nit, based on the logic I'm seeing

Reply:
Yeah, in future we would probably also filter based on chunk endorsements.